trace.go

```diff
@@ -24,6 +26,7 @@ import (
 
 var debug bool
 
+// EnableDebug turns on debugging mode, that causes Fatalf to panic
```

Comment:
I prefer not to export global shared state if it's not needed. Function gives us more options if we are about to change implementation, etc.